fenicsxconcrete/finite_element_problem/linear_elasticity.py

```diff
@@ -68,11 +68,16 @@ def sigma(self, u):
     def solve(self, t=1.0):        
         self.displacement = self.weak_form_problem.solve()
 
+        self.compute_residual()
```

Comment:
this is not always necessary, depending on the sensor that are used. this was also not properly implemented in fenics_concrete.
should we leave it like this? I would say the minimum should be a comment with a TODO and a new issue describing what needs to be done.

Reply:
I added a TODO mentioning that depends on sensors and added the abstract method to the ABC